tox.ini

```diff
@@ -10,5 +10,6 @@ commands =
     python manage.py --version
     python manage.py check
     black -l 80 --check .
-    django-admin compilemessages
+    django-admin compilemessages --ignore .tox
+    django-admin compilemessages --ignore .venv
```

Comment:
This will run `compilemessages` twice! Can you use the `cd wcivf && python ../manage.py compilemessages -l cy`?